app/views/partials/_landing_nav.html.erb

```diff
@@ -1,5 +1,9 @@
-<nav class="landing-nav">
+<%# <nav class="landing-nav">
   <form action="/users/sign_in" method="get">
     <button class="btn btn-white masthead-text login">Login</button>
   </form>
+</nav> %>
```

Comment:
Do we want this commented out code here?

Reply:
removed